presto-docs/src/main/sphinx/connector/hive.rst

```diff
@@ -210,6 +210,39 @@ Property Name                                      Description
  ``hive.metastore.catalog.name``                   Specifies the catalog name to be passed to the metastore.
 ================================================== ============================================================ ============
 
+Avro Configuration Properties
+-----------------------------
+
+When querying or creating Avro-formatted tables via the Hive connector, you may need to supply or override the Avro schema. In addition, Hive Metastore (especially Hive 3.x) must be configured to read storage schemas for Avro tables. The following properties are relevant:
+
+======================================================== ============================================================================== ======================================================================================
+Property Name                                            Description                                                                    Default
+======================================================== ============================================================================== ======================================================================================
+``avro_schema_url``                                      URL or path (HDFS, S3, HTTP, etc.) to the Avro schema file for                 None (must be specified if Metastore does not provide or you need to
+                                                         reading an Avro-formatted table. If specified, Presto will fetch               override schema)
+                                                         and use this schema instead of relying on any schema in the
+                                                         Metastore.
+``avro_schema_literal``                                  Inline JSON literal of the Avro schema to use when reading an                  None
```

Comment:
Usually, we only document table properties that are supported while creating a table in Presto.  As far as I am aware, `avro_schema_literal` property is not supported in Presto.